x-pack/server/lib/constants.js

```diff
@@ -6,3 +6,16 @@
 
 export const XPACK_DEFAULT_ADMIN_EMAIL_UI_SETTING = 'xPack:defaultAdminEmail';
 export const XPACK_INFO_API_DEFAULT_POLL_FREQUENCY_IN_MILLIS = 30001; // 30 seconds
+export const RANKED_LICENSE_TYPES = [
+  'basic',
+  'standard',
+  'gold',
+  'platinum',
+  'trial',
+];
+export const LICENSE_STATUS = {
+  UNAVAILABLE: 'UNAVAILABLE',
+  INVALID: 'INVALID',
+  INACTIVE: 'INACTIVE',
```

Comment:
What do you think of naming this to `EXPIRED` instead? An "expired" license is something I've heard of, whereas with "inactive" I always have to think about it for a second to figure out that it means the license is expired.